src/blocks/Card.tsx

```diff
@@ -21,6 +21,7 @@ export interface CardProps extends HTMLAttributes<HTMLDivElement> {
   heading_alignment?: HeadingAlignment
   body?: string
   image?: CardImage
+  image_srcset_medium?: string
```

Comment:
I think this prop should just be called `image_srcset` here… Why the `medium` if there are no others? Or maybe help me understand your thought process here @ktryndchrs ?

Actually, shouldn't the `image`, of type CardImage, have a `srcset` prop, making this `image_srcset_medium` prop useless?

Reply:
Bien sûr, il y a une raison logique derrière!  La voici :

https://user-images.githubusercontent.com/416462/181380147-00602f1c-4fbd-4b16-9b4f-723686ad4696.mp4

Reply:
Tous équivalents:
```
<Card image={...} image_srcset_medium={...} />
<Card image={img} image_srcset={ss} />
<Card image={img} srcset={ss} />
<Card image={img} xyz={ss} />
```

Dans tous les cas, la valeur du prop est acheminé vers un prop de `<Media>`. 

L'autre option serait:
```
<Card image={{...img, srcset: ss }} />
```